style.css

```diff
@@ -407,24 +430,43 @@ body {
   background-image: url("./images/bg_skills.png");
 }
 
-.flex-form-section {
+/* Footer section */
+
+footer {
+  margin-top: 10%;
```

Comment:
Let's try to reduce the margin a little bit to make it look closer to the other sections 👍